src/components/booking/ChooseSeats.vue

```diff
@@ -0,0 +1,126 @@
+<script setup lang="ts">
+import { computed } from "vue";
+import { storeToRefs } from "pinia";
+import { useReservationStore } from "@/stores/reservation.js";
+import BaseButton from "@/components/global/BaseButton.vue";
+
+interface chosenSeat {
```

Comment:
We should write our types in `PascalCase` to let them stand out and be consistent 😄 
```suggestion
interface ChosenSeat {
```